starters/cra-rxjs-styled-components/package.json

```diff
@@ -9,12 +9,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "autoprefixer": "^10.4.7",
+    "postcss": "^8.4.14",
```

Comment:
We should not be installing tailwind. This project is intended to only use Styled Components. I linked the original Tailwind article as a guide for what CSS you could use to make this show up quickly since Tailwind translates to plain CSS pretty easily.